scripts/taxonomy/sesar_indexing_test_records.json

```diff
@@ -0,0 +1,3166 @@
+{
```

Comment:
Could you delete this file and replace the lines in `isamples_inabox/integration_tests/test_model_values.json` please?